RULES.md

```diff
@@ -0,0 +1,37 @@
+# Rules
+
+A number of rules are provided in this pack on a number of triggers defined in the `stackstorm_send_notifications` pack
+to handle rabbitmq message commands.
+
+## Hypervisor Rules
+
+| Rule      | Description |
+| ----------- | ----------- |
+| `rabbitmq.hypervisor.disable.service.rule` | matches trigger `rabbit.message` with `message_type=HYPERVISOR_DISABLE_SERVICE` and calls `hypervisor.service.disable` using `stackstorm_send_notification.rabbit.execute`|
```

Comment:
IMO the names should be self obvious instead of us having to write documentation to make up for it:

`rule.on_hypervisor_disable` would remove the need to document this and make it obvious what the rule does if it was in a list without a description

Reply:
I disagree - I'd rather have documentation even if it should be obvious